hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -151,7 +151,25 @@
       reset_connections: {rst_ni: "lc"},
       type: "eflash",
       base_addr: "0x20000000",
-      size: "0x80000" },
+      size: "0x80000",
+      inter_signal_list: [
+        { struct: "flash",    // flash_req_t, flash_rsp_t
```

Comment:
these comments are fine for now (since we're in testing-out mode) but we should discourage users from having to explain the details, but instead make sure the expansion of the constructs is well documented so it is "obvious".